tests/integration/test_system_containers_install.sh

```diff
@@ -138,6 +138,9 @@ ${ATOMIC} pull --storage ostree docker:atomic-test-runonce:latest
 ${ATOMIC} install --system --name=${NAME} --set RECEIVER=Pluto atomic-test-runonce:latest > ${WORK_DIR}/ps.out
 assert_matches "HI Pluto from Saturn" ${WORK_DIR}/ps.out
 
+${ATOMIC} run --storage ostree --set RECEIVER=Pluto atomic-test-runonce:latest echo Hello World > ${WORK_DIR}/ps.out
```

Comment:
Atomic run has no `--set` option right? I believe this is causing ci failures.

Reply:
ouch, I assumed it had.  I added a patch to implement it as it is very useful for running containers.